vcs/none.go

```diff
@@ -0,0 +1,46 @@
+package vcs
+
+import (
+	"log"
+	"os/exec"
+	"path/filepath"
+)
+
+func init() {
+	Register(newNoneVcs, "none")
+}
+
+type NoneVcsDriver struct{}
+
+func newNoneVcs(b []byte) (Driver, error) {
+	return &NoneVcsDriver{}, nil
+}
+
+func (g *NoneVcsDriver) HeadRev(dir string) (string, error) {
+    return "n/a", nil
+}
+
+func (g *NoneVcsDriver) Pull(dir string) (string, error) {
+	return g.HeadRev(dir)
+}
+
+func (g *NoneVcsDriver) Clone(dir, url string) (string, error) {
+	par, rep := filepath.Split(dir)
+	cmd := exec.Command(
+		"cp",
+		"-r",
+		url[7:],
```

Comment:
This seems a little odd...hardcoding `7` here strips off the `file://` I assume? I'd prefer to do a string replace I think.